app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/Platform.scala

```diff
@@ -6,12 +6,71 @@ import java.sql.Timestamp
 import io.circe._
 import io.circe.generic.JsonCodec
 import io.circe.syntax._
+import io.circe.generic.semiauto._
+import cats.syntax.either._
+
 
-@JsonCodec
 case class Platform(
   id: UUID,
   name: String,
-  settings: Json,
+  publicSettings: Platform.PublicSettings,
   isActive: Boolean,
-  defaultOrganizationId: Option[UUID]
+  defaultOrganizationId: Option[UUID],
+  privateSettings: Platform.PrivateSettings
 )
+
+object Platform {
+
+  def tupled = (Platform.apply _).tupled
+
+  case class PublicSettings(
```

Comment:
In scenes, we threw our helper case classes into the outermost scope of the package. I like this a lot better -- `Platform.PublicSettings` gives way more context than some `PublicSettings` case class floating around :tada: good choice

Reply:
e.g., `SceneFilterFields` could become `Scene.FilterFields` and I think improve clarity -- by tying the scope of the filter fields to the scope of the object that it's going to filter (https://github.com/raster-foundry/raster-foundry/blob/develop/app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/Scene.scala#L18-L23)

Reply:
(You're not responsible for changing scenes -- I'm just pointing out how what you did here is better than what I did over there a while ago)